include/GafferOSL/ShadingEngine.h

```diff
@@ -53,7 +54,15 @@ class ShadingEngine : public IECore::RefCounted
 		ShadingEngine( const IECore::ObjectVector *shaderNetwork );
 		~ShadingEngine();
 
-		IECore::CompoundDataPtr shade( const IECore::CompoundData *points ) const;
+		struct Transform
+		{
+			Imath::M44f fromObjectSpace;
+			Imath::M44f toObjectSpace;
+		};
+
+		typedef std::map<OIIO::ustring, Transform> Transforms;
```

Comment:
Once we've removed the OIIO header, we'll need to use either `string` or `IECore::InternedString` here.